admin_guide/aggregate_logging.adoc

```diff
@@ -115,47 +131,56 @@ contents:
 ----
 <1> The name for the master that will be used during logging.
 <2> The IP or a DNS resolvable name used to access the master.
+====
 
-=== Enabling fluentd
+. Enable *fluentd*:
 
-[source,sh]
+====
 ----
 # systemctl enable td-agent
 # systemctl start td-agent
 ----
+====
 
-TIP: Any errors will be logged in the *_/var/log/td-agent/td-agent.log_* file.
+[TIP]
+====
+Any errors are logged in the *_/var/log/td-agent/td-agent.log_* file.
+====
 
 === Optional Method to Verify Working Nodes
 
 You can optionally set up the master to be the aggregator to test and verify
 that the nodes are working properly.
 
-*Installing fluentd (td-agent) on the Master*
-
-[source,sh]
+. Install *fluentd* (*td-agent*) on the master:
++
+====
 ----
 # export RPM=td-agent-2.2.0-0.x86_64.rpm
-# curl http://packages.treasuredata.com/2/redhat/7/x86_64/$RPM -o /tmp/$RPM
+# curl http://packages.treasuredata.com/2/redhat/7/x86_64/$RPM \
+    -o /tmp/$RPM
 # yum localinstall $RPM
 # mkdir -p /etc/td-agent/config.d
 # chown td-agent:td-agent /etc/td-agent/config.d
 ----
+====
 
-Ensure port 24224 is open on the master's firewall to allow the nodes access.
-
-*Configuring fluentd to Aggregate Container Logs*
-
-Add the following line to the *_/etc/td-agent/td-agent.conf_* file:
+. Ensure port *24224* is open on the master's firewall to allow the nodes
+access.
```

Comment:
@adellape Not too important if we're thinking that our reader is an admin, but should there be an example command for this?